content/kubermatic/master/tutorials_howtos/networking/ipam/_index.en.md

```diff
@@ -0,0 +1,157 @@
++++
+title = "Multi-Cluster IP Address Management (IPAM)"
+date = 2022-07-26T14:45:00+02:00
++++
+
+Feature responsible for automating the allocation of ranges of IP addresses per-user-cluster, based on a predefined configuration ([IPAMPool](#input-resource-ipampool)) per datacenter that defines the pool subnet and the allocation size. The user cluster allocated ranges are available in the [KKP Addon](#kkp-addon-template-integration) `TemplateData`, so it can be used by various applications / Addons running in the user cluster.
```

Comment:
"the allocation of ranges of IP addresses per-user-cluster" -> "the allocation of IP address ranges per user-cluster"